schemas/nrjmx.yml

```diff
@@ -76,3 +76,32 @@
       dest: "{dest_prefix}linux/yum/amazonlinux/{os_version}/{arch}/"
       os_version:
         - 2
+        - 2022
+
+- src: "{app_name}-{version}-1.noarch.rpm"
```

Comment:
In understand adding/removing versions from the existing entries, but why do we need to add new full entries for this?

Reply:
this is replicating what is being done with the [Agent](https://github.com/newrelic/infrastructure-agent/blob/f9f197ab5bfd24c8d5c2781ee3c1c2f7883b99f9/build/upload-schema-linux-rpm.yml).

I moved the PR to draft since I think that we could address it once the Agent packages are 🧹